aiida/parsers/plugins/quantumespresso/raw_parser_pw.py

```diff
@@ -1245,6 +1250,24 @@ def parse_pw_text_output(data, xml_data={}, structure_data={}, input_dict={}):
                 parsed_data['wall_time_seconds'] = convert_qe_time_to_sec(time)
             except ValueError:
                 raise QEOutputParsingError("Unable to convert wall_time in seconds.")
+
+        elif any(bool_time) and 'WALL' in line:
+            bindex = bool_time.index(True)
+            bstring = times_strings[bindex] + '_time'
+            bstring_seconds= times_strings[bindex] + '_time_seconds'
+            try:
+                time = line.split('CPU')[1].split('WALL')[0]
+                parsed_data[bstring] = time
+            except Exception:
+                error_string = 'Error while parsing ' + times_strings[bindex]
+                parsed_data['warnings'].append(error_string)                
+            try:
+                parsed_data[bstring_seconds] = convert_qe_time_to_sec(time)
+            except ValueError:
+                error_string = 'Unable to convert ' + times_strings[bindex] + ' seconds'
+                raise QEOutputParsingError(error_string)
```

Comment:
Here I would just add a warning to the `parsed_data['warnings']` rather than raising an exception.